base/applications/notepad/lang/it-IT.rc

```diff
@@ -25,7 +25,7 @@ BEGIN
     POPUP "&File"
     BEGIN
         MENUITEM "&Nuovo\tCtrl+N", CMD_NEW
-        MENUITEM "New &Window\tCtrl+Shift+N", CMD_NEW_WINDOW
+        MENUITEM "Nuova f&inestra\tCtrl+Shift+N", CMD_NEW_WINDOW
```

Comment:
are you sure, that 2k3 does use &i here? That is not impossible, but would be rather uncommon, because MS does usually not recommend using &i in their guidelines. Check 2k3 please!

Reply:
There is no new window in win 2k3 notepad.  Only the modern version has it.
![image](https://github.com/user-attachments/assets/92520fce-cadd-4c3e-b932-3c864dfd1311)
![image](https://github.com/user-attachments/assets/8b8358b1-4a5b-42a6-bcd8-22c789e3920f)

Reply:
Then you can ofc seek inspiration from the newer versions accelerator. You can watch the accelerators on more modern Windows versions by pressing Alt afaik. By default it seems to hide them.

Reply:
Found it. It's the f.